client/global.h

```diff
@@ -66,6 +66,8 @@ extern int g_disable_mining;
 //Test network and main network have different types of the block headers, so blocks from different networks are incompatible
 extern enum xdag_field_type g_block_header_type;
 
+extern xdag_time_t g_apollo_fork_time;
```

Comment:
There is no need g_apollo_fork_time at all.

Reply:
we use nmain as hegiht to fork apollo feature, also we need this var to record the apollo fork time，because block command use xdag_print_block_info to print earning amount, when user input old block address on store loaded, we don't have nmain on that time, so print right amount of earning must use this time.